source/Pages/monitoring.rst

```diff
@@ -0,0 +1,46 @@
+.. _monitoring-spider:
+*****************
+Monitoring Spider
+*****************
+
+There are two types of monitoring dashboards for :abbr:`Spider (Symbiotic Platform(s) for Interoperable Data Extraction and Redistribution)`, a `public section <https://jobsview.grid.surfsara.nl/grafana/d/i289WluZz/spider?orgId=1>`_ visible for everyone and a section visible only to `authenticated Spider users <https://monitor.spider.surfsara.nl/grafana/dashboards>`_.
+Both are shortly described below. First some general properties and settings are covered. 
+
+At the top of each of these dashboards is a time selection drop down to change the time range of the graph on the x-axis, along with a refresh button and refresh timer.
+Below these buttons are the actual properties of Spider being shown, which is either a single number of a property, for example current RAM usage, or a time-dependent graph of a property, for example CPU usage over time. If a panel has a legend, a single legend entry can be selected by clicking on it, which filters out the rest. It can be deselected by pressing it again.
+
+Public dashboard
+================
+
+The public dashboard can be found `here <https://jobsview.grid.surfsara.nl/grafana/d/i289WluZz/spider?orgId=1>`_ and shows 4 views of running jobs, pending jobs, core usage and queue times for different projects.
```

Comment:
small note: jobsview does not show actual core usage by job processes in terms of what a user may naively expect, what it in fact shows is the total number cores reserved by slurm for all running jobs of a project. In both the documentation and in the grafana we may consider using the word reserved (or another better word if it exists) instead of used/usage